boards/common/remote/Makefile.include

```diff
@@ -11,6 +11,9 @@ ifneq (,$(PORT_BSL))
 endif
 
 ifeq ($(PROGRAMMER),cc2538-bsl)
+  PROG_BAUD ?= 115200
```

Comment:
Why 115200?

Reply:
This is the value that was used initially. This is why.

Reply:
Ah sorry didn't see the previous line.

Reply:
I think it didn't show up because the lines were not removed previously, so not highlighted in Github interface ;)